stack.yaml

```diff
@@ -48,7 +48,7 @@ packages:
   extra-dep: true
 - location:
     git: https://github.com/serokell/time-warp-nt.git
-    commit: 00f9911afa62ff6af5d53d4c9bb2d5c8aa72ec57 # master
+    commit: 7ec342c59823d9890836038122ebad7eb3f187ca # master
```

Comment:
This is what it's all about really. Other changes are to catch up to the new time-warp-nt revision.